src/Loading/babylon.sceneLoader.ts

```diff
@@ -71,34 +82,40 @@
             }
 
             var extension = sceneFilename.substring(dotPosition, queryStringPosition).toLowerCase();
+            return SceneLoader._registeredExtensions[extension];

```

Comment:
okay, yeah, that was what it was doing before. if the extension is unknown, return the Babylon loader.